brave/brave-core
# Refactored Brave Ads in preparation for merging Publisher Ads

Resolves https://github.com/brave/brave-browser/issues/8353

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm Brave ad notifications are shown
- Confirm EventLog's, i.e. blurred are logged to the console log under `AdsService Event Log:`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
